README.md

```diff
@@ -176,7 +176,7 @@ plan myapp::patch (
   run_plan('reboot', $servers, reconnect_timeout => 300)
 
   # Check the status of the service
-  return run_task('service', $nodes, {
+  return run_task('service', $targets, {
```

Comment:
Oh lol I think this is a typo...it looks like it should be `$servers` -  I don't think nodes or targets is defined there.